Karma peer dependency problems on npm install

So this might be more of an issue with Karma rather than MEAN but I thought I should mention it.

In package.json you list devDependencies to include all the karma plugins that are now include by default in Karma. However, Karma requires these plugins as "*", the latest version. Which means that there can be dependency issues.

When running `npm install` I get these errors because of what I mentioned above

```
npm ERR! peerinvalid The package karma-requirejs does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer karma@0.10.10 wants karma-requirejs@~0.2.0

npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.24
npm ERR! code EPEERINVALID
npm ERR! not ok code 0
```

I'd recommend removing all the Karma plugins from the packages.json as they come with Karma anyway and shouldn't break anything else (to the best of my knowledge)

Cheers

--- a/package.json
+++ b/package.json
@@ -50,6 +50,7 @@
     "grunt-concurrent": "latest",
     "grunt-mocha-test": "latest",
     "karma": "~0.10.4",
+    "karma-coverage": "~0.1.0",
     "supertest": "0.8.2",
     "should": "2.1.1"
   }

